benchexec/test_analyze_run_result.py

```diff
@@ -25,14 +25,14 @@
 normal_result = ProcessExitCode(raw=0, value=0, signal=None)
 
 
-class TestResult(unittest.TestCase):
-    @classmethod
-    def setUpClass(cls):
-        cls.longMessage = True
-        logging.disable(logging.CRITICAL)
+@pytest.fixture(scope="class")
+def setup_class():
+    logging.disable(logging.CRITICAL)
```

Comment:
Wasn't the outcome of the previous discussion that we can get rid of all the logging config in tests?

Reply:
yes, I must've forgot to make the change